sunpy/sun/models.py

```diff
@@ -198,3 +208,43 @@ def differential_rotation(duration: u.s, latitude: u.deg, *, model='howard', fra
         rotation -= 0.9856 * u.deg / u.day * duration
 
     return Longitude(rotation.to(u.deg))
+
+
+
+_MODEL_DATA_DIR = pathlib.Path(__file__).parent.absolute()/"data"
+
+# Dictionary mapping model names to file paths
+_CHROMOSPHERE_MODELS = {
+    "avrett_loeser_2008": _MODEL_DATA_DIR / "chromosphere_avrett_Loeser_2008_model.ecsv"
+}
+
+# Cache to store loaded models
+_chromosphere_cache = {}
```

Comment:
```suggestion
_MODEL_DATA_DIR = pathlib.Path(__file__).parent.absolute()/"data"
_CHROMOSPHERE_MODELS = {
    "avrett_loeser_2008": _MODEL_DATA_DIR / "chromosphere_avrett_Loeser_2008_model.ecsv"
}
_MODEL_CACHE = {}
```
These should be at the top of the file,. under the all